### PR TITLE
Setup monorepo

### PR DESCRIPTION
**Pull request by fondation451**
Issue: https://github.com/fondation451/label/issues/27

Here is a proposition for a monorepo architecture.
Actually, it seems to be the best way, a single package.json is to messy.

I have normalized the scripts between the different package.json.

### CODE REVIEW

File: packages/backend/Dockerfile [hunk 1]
```diff
@@ -7,7 +7,6 @@ WORKDIR /usr/src/app
 
 # Copy context files
 COPY package.json ./
-COPY yarn.lock ./
```
Comment (BenoitSerrano) [inline packages/backend/Dockerfile:10]: Why did you remove it ?

File: packages/client/Dockerfile [hunk 2]
```diff
@@ -5,11 +5,10 @@ FROM node:14-alpine
 WORKDIR /usr/src/app
 
 # add `/app/node_modules/.bin` to $PATH
-ENV PATH /app/node_modules/.bin:$PATH
+ENV PATH /usr/src/app/node_modules/.bin:$PATH
 
 # install app dependencies
 COPY package.json ./
-COPY yarn.lock ./
```
Comment (BenoitSerrano) [inline packages/client/Dockerfile:12]: Why did you remove it?

**Comment by fondation451** [comment]
@BenoitSerrano 
Because, with the monorepo structure, there is only one `yarn.lock` which is at the root of the project. It is outside the scope of the Dockerfile, so it could not be copied.
I don't think it is important, because when we create the docker, we should not assume that a `yarn install` as been made.

**Comment by fondation451** [comment]
@BenoitSerrano 
J'ai remis le yarn.lock dans les Dockerfile.

L'architecture en monorepo fonctionne même avec les dockers.
J'ai pas du tout re-travaillé mes commits, donc il vaut mieux que tu regardes le diff global.

On peut en discuter demain.

**Comment by fondation451** [comment]
@BenoitSerrano  J'ai commenté quelque chose dans le dockerfile du client car ça faisait bugué.

A quoi ça servait ?

File: docker/client/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+services:
+  client:
+    container_name: client
+    stdin_open: true
+    build:
+      context: ../../
+      dockerfile: docker/client/Dockerfile
+    #    volumes:
+    #      - ".:/usr/src/app"
```
Comment (BenoitSerrano) [inline docker/client/docker-compose.yml:11]: I think it deactivates the hot reloading
